doc/develop/conformance_rules.md

```diff
@@ -115,14 +114,12 @@ MyClass.prototype.method = function(arg) {
 }
 ```
 
-
-{: #logger}
-### goog.debug.Logger 
+### goog.debug.Logger {#logger}
```

Comment:
This will break our documentation.  The problem is that internal markdown and github-compatible markdown have mutually incompatible syntax for this, so we were using a MOE transformer to fix this.

It's probably fine to take this for now and we can look into fixing it up with some travis post-processing ASAP.